tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -61,29 +84,19 @@ def test_get_extension():
 
 def test_buckets_logic():
     stacktraces = [
-        "<anonymous>",  # Garbage
-        "<frozen importlib._bootstrap>",
-        "[native code]",
-        "O$t",
-        "/foo/bar/baz",  # no extension
-        "README",  # no extension
-        "backburner.js",
         "app://foo.js",
         "./app/utils/handleXhrErrorResponse.tsx",
         "getsentry/billing/tax/manager.py",
-        "/gtm.js",  # Top source; starts with backslash
-        "ssl.py",
-    ]
+    ] + UNSUPPORTED_FRAME_FILENAMES
     buckets = stacktrace_buckets(stacktraces)
     assert buckets == {
         "./app": [FrameFilename("./app/utils/handleXhrErrorResponse.tsx")],
-        "NO_TOP_DIR": [FrameFilename("/gtm.js"), FrameFilename("ssl.py")],
         "app:": [FrameFilename("app://foo.js")],
         "getsentry": [FrameFilename("getsentry/billing/tax/manager.py")],
     }
 
 
-class TestFrameFilename(TestCase):
+class TestFrameFilename:
```

Comment:
This turns this group of tests into unit tests rather than requiring initializing the DB when we don't have to.